### DISCUSSION TITLE
Issue with [@types/google.accounts] use_fedcm_for_prompt

### DISCUSSION THREAD

**Discussion by ftirellisplit** [DefinitelyTyped/DefinitelyTyped#66876] [Issues with a @types package]
As part of the FedCM Google beta program a new prop (use_fedcm_for_prompt) is been added to the IdConfiguration. I leave you the link to the doc:

https://developers.google.com/identity/gsi/web/reference/js-reference#IdConfiguration

[a comment by an automated account is not shown]

**Comment by PartyLich** [comment]
@ftirellisplit https://github.com/DefinitelyTyped/DefinitelyTyped/pull/66778 might cover this; feel free to open a PR if not!
